docs/source/deployorderer/ordererdeploy.md

```diff
@@ -0,0 +1,299 @@
+
+At a minimum, you need to configure the following parameters:
+- `General.ListenAddress` - Hostname that the ordering node listens on.
+- `General.ListenPort` - Port that the ordering node listens on.
+- `General.TLS.Enabled: true` - Server-side TLS should be enabled in all production networks.
+- `General.TLS.PrivateKey ` - Ordering node private key from TLS CA.
+- `General.TLS.Certificate ` - Ordering node signed certificate (public key) from the TLS CA.
+- `General.TLS.RootCAS` - This value should be unset.
+- `General.Cluster.ClientCertificate` - Ordering node signed certificate (public key) from the TLS CA.
+- `General.Cluster.ClientPrivateKey` - Ordering node private key from TLS CA.
+- `General.BoostrapMethod:file` - Start ordering service with a system channel.
+- `General.BootstrapFile` - Path to and name of the genesis block `.pb` file for the ordering service system channel.
+- `General.LocalMSPDir` - Path to the ordering node MSP folder.
+- `General.LocalMSPID` - MSP ID of the ordering organization as specified in the channel configuration.
+- `FileLedger.Location` - Location of the orderer ledger on the file system.
+
+## Start the orderer
+
+Make sure you have set the value of the `FABRIC_CFG_PATH` to be the location of the `orderer.yaml` file relative to where you are invoking the orderer binary. For example, if you run the orderer binary from the `fabric/bin` folder, it would point to the  `/config` folder:
+    ```
+    export FABRIC_CFG_PATH=../config
+    ```
+
+After `orderer.yaml` has been configured and your deployment backend is ready, you can simply start the orderer node with the following command:
+
+```
+cd bin
+./orderer start
+```
+
+When the orderer starts successfully, you should see a message similar to:
+
+```
+INFO 019 Starting orderer:
+INFO 01a Beginning to serve requests
+```
+
+You have successfully started one node, you now need to repeat these steps to configure and start the other two orderers. When a majority of orderers are started, a Raft leader is elected. You should see something similar to the following output:
+```
+INFO 01b Applied config change to add node 1, current nodes in channel: [1] channel=syschannel node=1
+INFO 01c Applied config change to add node 2, current nodes in channel: [1 2] channel=syschannel node=1
+INFO 01d Applied config change to add node 3, current nodes in channel: [1 2 3] channel=syschannel node=1
+INFO 01e raft.node: 1 elected leader 2 at term 11 channel=syschannel node=1
+INFO 01f Raft leader changed: 0 -> 2 channel=syschannel node=1
+```
+
+## Next steps
+
+Your ordering service is started and ready to order transactions into blocks. Working together to form the ordering service, the orderers receive transactions from application clients and package the transactions into a well-defined sequence of blocks. These blocks form the blockchain and are ultimately distributed to all peers on the channel to be committed to their ledger.
```

Comment:
Actually, at this point, we are not ready to receive transactions from application clients, since we've only created the system channel, we haven't crated application channels yet.